cli/concept.md

```diff
@@ -0,0 +1,157 @@
+# Overview
+The Amplify cli enables front-end developers to easily set up the backend resources in the cloud. <br/>
+It is designed to work side by side with the [Amplify](https://github.com/aws-amplify/amplify-js) library. Official resource categories provided by the cli can be easily consumed by the corresponding Amplify library modules. <br/>
+The Amplify cli is written in Node.js. It has a pluggable architecture and can be easily extended with additional functionalities.
+ 
+***
+
+# Architecture
+![](https://github.com/UnleashedMind/docs/blob/master/cli/images/AmplifyCliConcept.jpg)
+## Cli core and plugins
+The Amplify cli uses `gluegun`. It is highly modularized.  <br/>
+The cli core provides the pluggable platform, most of the cli functionalities are delivered by plugins. <br/> 
```

Comment:
replace with "The CLI core provides the pluggable platform, and most of the CLI category functions are implemented as plugins."